src/components/Contacts/StarContactIcon/StarContactIcon.tsx

```diff
@@ -1,26 +1,25 @@
-import { makeStyles } from '@material-ui/core';
+import { styled } from '@material-ui/core';
 import React from 'react';
 import { StarBorderOutlined } from '@material-ui/icons';
 import StarIcon from '@material-ui/icons/Star';
 import theme from '../../../theme';
 
-const useStyles = makeStyles(() => ({
-  contactStar: {
-    width: '24px',
-    height: '24px',
-    color: theme.palette.secondary.main,
-  },
-}));
-
 interface Props {
   hasStar: boolean;
 }
 
+const StarFilled = styled(StarIcon)(({}) => ({
```

Comment:
Are you guys using styled option instead of makeStyles now?

Reply:
Ya I think we prefer this syntax. Ties styles to a specific component directly and makes it clear when something is no longer used and can be deleted.